firebase/functions/index.js

```diff
@@ -4,158 +4,132 @@
 
 const functions = require('firebase-functions');
 const admin = require('firebase-admin');
-const {PubSub} = require(`@google-cloud/pubsub`);
+const { PubSub } = require(`@google-cloud/pubsub`);
 const pubsub = new PubSub();
 
-const EXPIRY_MS = 1000 * 60 * 60 * 24;
-
 admin.initializeApp(functions.config().firebase);
 const db = admin.firestore();
 
-function deleteRun(port, port_doc, runid) {
```

Comment:
where is this functionality going?

Reply:
It's not needed so I deleted it. It's the same with clearport function.

Reply:
Both the deleterun and clearport existed for a reason, so not sure if the functionality has been moved elsewhere, or has been obsoleted in some way?

Reply:
Yes clearport result is now obsolete with my changes and there's no need to delete any previous run data with the way the data is queried/ structured.